src/dialect/snowflake.rs

```diff
@@ -245,6 +245,14 @@ impl Dialect for SnowflakeDialect {
                     .map(|p| Some(ColumnOption::Policy(ColumnPolicy::ProjectionPolicy(p)))))
             } else if parser.parse_keywords(&[Keyword::TAG]) {
                 Ok(parse_column_tags(parser, with).map(|p| Some(ColumnOption::Tags(p))))
+            } else if parser.parse_keywords(&[Keyword::COMMENT]) {
+                let next_token = parser.next_token();
+                match next_token.token {
+                    Token::DollarQuotedString(value, ..) => {
+                        Ok(Ok(Some(ColumnOption::Comment(value.value))))
+                    }
+                    _ => Err(ParserError::ParserError("not found match".to_string())),
```

Comment:
hmm the error message sounds like it'd be confusing what we mean by 'not found match'.

Left a comment below about reusing the parsing code, we probably want to reuse it here as well (any reason not to cover the single quote case here)?

Reply:
That makes sense. I have moved this code to the parser level.